draft-ietf-quic-transport.md

```diff
@@ -2152,11 +2152,10 @@ receive packets without first having sent a packet on that path. Effective NAT
 traversal needs additional synchronization mechanisms that are not provided
 here.
 
-An endpoint MAY include PATH_CHALLENGE and PATH_RESPONSE frames that are used
-for path validation with other frames.  In particular, an endpoint can pad a
-packet carrying a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU)
-discovery (see {{pmtud}}), or an endpoint can include a PATH_RESPONSE with its
-own PATH_CHALLENGE.
+An endpoint MAY include other frames with the PATH_CHALLENGE and PATH_RESPONSE
+frames used for path validation.  In particular, an endpoint can include
+PADDING with a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU) discovery
+(see {{pmtud}}), and/or include a PATH_RESPONSE with its own PATH_CHALLENGE.
 
 When probing a new path, an endpoint might want to ensure that its peer has an
```

Comment:
This paragraph is very duplicative of a paragraph in "Probing a new path" under connection migration, so I think we should drop one of them, but I'm not clear on which.

Reply:
The latter can probably go.

Reply:
Done.